src/ensembl/src/content/app/browser/track-panel/track-panel-bar/TrackPanelBarIcon.tsx

```diff
@@ -39,19 +41,22 @@ const TrackPanelBarIcon: FunctionComponent<TrackPanelBarIconProps> = memo(
       setToggleState(newToggleState);
     }, [props.iconConfig.name, toggleState]);
 
-    const getViewIcon = () => {
+    const getViewIconStatus = () => {
       const { iconConfig, trackPanelModalView } = props;
 
       return iconConfig.name === trackPanelModalView
-        ? iconConfig.icon.selected
-        : iconConfig.icon.default;
+        ? ImageButtonStatus.HIGHLIGHTED
+        : ImageButtonStatus.ACTIVE;
     };
 
     return (
       <dt className={styles.barIcon}>
-        <button onClick={toggleModalView}>
-          <img src={getViewIcon()} alt={props.iconConfig.description} />
-        </button>
+        <ImageButton
+          buttonStatus={getViewIconStatus()}
+          description={props.iconConfig.description}
+          onClick={toggleModalView}
+          image={props.iconConfig.icon}
+        />
```

Comment:
Here's what I am currently seeing on the production site:

![Screenshot 2019-03-28 at 15 27 04](https://user-images.githubusercontent.com/6834224/55170313-3b05d080-516e-11e9-86b8-576ab49c805e.png)

And here's what I am seeing in this branch:

![Screenshot 2019-03-28 at 15 27 20](https://user-images.githubusercontent.com/6834224/55170350-478a2900-516e-11e9-8044-d6f12fe73379.png)

The styles are off.

Reply:
Seems to be working fine for me. Can you clear the cache and try again, please?

Reply:
Yep, looks fine now.